book/website/collaboration/oss-sustainability/oss-sustainability-challenges.md

```diff
@@ -0,0 +1,75 @@
+
+
+# Sustainability of Open Source Projects
+
+Early-stage projects often reach a point when funding has run out for a project that they have spent months or years developing, and they can face a daunting array of obstacles on the road to turning an academic or volunteer-led project into a robust, sustainable enterprise that will outlive its original funding source. 
+In this chapter we will outline the factors that should be considered when you want to turn a project into Open Source Software (OSS) with a self-sustaining organisation, and we will explore how different projects have achieved sustainability through a variety of organisational structures.
+
+## The big misconception about Open Source
+```{figure} ../../figures/oss-sustainability-pathways-problem.jpg
+---
+name: oss-sustainability-pathways-problem
+alt: A researcher is torn between taking the path of private commercialisation or the path of open source.
+---
+This image was created by Scriberia for The Turing Way community and is used under a CC-BY 4.0 licence for reuse. Zenodo. DOI 10.5281/zenodo.3332807
+```
+
+One of the most common beliefs about Open Source is that it simply means "free" and therefore an open source project can never generate revenue or profit.
+The [definition](https://the-turing-way.netlify.app/reproducible-research/open.html) used by The Turing Way states that OSS should:
+>Be publicly available: It is difficult to use and benefit from knowledge hidden behind barriers such as passwords and paywalls.
+>
```

Comment:
Something maybe about the header hierarchy (this and the below being ##) makes it look like there's a big section missing here